### PR TITLE
968 new command reset input

### PR DESCRIPTION
**Pull request by RobertPoienar**

### CODE REVIEW

**Comment by RobertPoienar** [comment]
@gomboskriszta15 @robert-altom @ReichEdvin @rucindrea

**Comment by robert-altom** [comment]
Looks good to me.

File: Docs/source/pages/commands.md [hunk 1]
```diff
@@ -1782,6 +1782,79 @@ Simulates device rotation action in your game.
 
 ```
 
+#### ResetInput
+
+Clear all active input actions simulated by AltTester.
```
Comment (iulianatodoran) [inline Docs/source/pages/commands.md:1787]: should be `Clears` instead of `Clear`

Comment (iulianatodoran) [reply Docs/source/pages/commands.md:1787]: I will make a separate ticket for this typo not to block the merge of this PR :D

File: Assets/AltTester/AltServer/NewInputSystem.cs [hunk 2]
```diff
@@ -252,17 +279,20 @@ internal static IEnumerator ClickElementLifeCycle(GameObject target, int count,
         internal static IEnumerator ClickCoordinatesLifeCycle(UnityEngine.Vector2 screenPosition, int count, float interval)
         {
             Mouse.MakeCurrent();
-            InputTestFixture.Set(Mouse.position, screenPosition, queueEventOnly: true);
+            UnityEngine.Debug.Log("flajkdklajslk11111: " + Mouse.leftButton);
```
Comment (gomboskriszta15) [inline Assets/AltTester/AltServer/NewInputSystem.cs:282]: You forgot this log here

File: Assets/AltTester/AltServer/NewInputSystem.cs [hunk 3]
```diff
@@ -289,7 +319,9 @@ internal static void KeyUp(KeyCode keyCode)
         internal static IEnumerator KeyPressLifeCycle(KeyCode keyCode, float power, float duration)
         {
             keyDownPower = power;
+            UnityEngine.Debug.Log("aodasdkal2222222a===> " + keyCode);
```
Comment (gomboskriszta15) [inline Assets/AltTester/AltServer/NewInputSystem.cs:322]: Same here

File: Assets/AltTester/AltServer/NewInputSystem.cs [hunk 4]
```diff
@@ -289,7 +319,9 @@ internal static void KeyUp(KeyCode keyCode)
...
             ButtonControl buttonControl = keyCodeToButtonControl(keyCode, power);
+            UnityEngine.Debug.Log("aodasdkala===> " + buttonControl.name);
```
Comment (gomboskriszta15) [inline Assets/AltTester/AltServer/NewInputSystem.cs:324]: And here

**Review by ReichEdvin — APPROVED** [review]
LGTM

**Review by iulianatodoran — APPROVED** [review]
I will approve this PR  instead of @robert-altom who only left an approval comment without and actual approve

**Pull request merged**